packages/timeline-state-resolver/src/devices/__tests__/vmixAPI.spec.ts

```diff
@@ -10,48 +9,50 @@ async function wait(time = 1) {
 	})
 }
 describe('vMixAPI', () => {
```

Comment:
Perhaps we should add a test in here that tests the functinoality in `BaseConnection._processIncomingData()`, 
especially things related to messages-split-over-multiple-lines?

Reply:
I've tweaked the behavior of the vMix mock so that it hits the messages split over multiple lines and messages split over multiple packets too, so I hope this is now handled.